.github/workflows/status-check-override.yaml

```diff
@@ -0,0 +1,92 @@
+# For pull requests that only modify documentation, individual CI events may forego running.
+# (This is done by each action/pipeline/event's config.) The itwinjs-core repo requires some
+# of those events to have finished successfully or neutrally to allow a pull request to be
+# merged.
+#
+# This config will fake successful status checks for such events when it detects
+# *only* documentation changes in a pull request.
+
+name: Skip Check
+
+on:
+  # workflow_dispatch:   # Allows you to run this workflow manually from the Actions tab
+  pull_request:
+    branches:
+      - master
+      - release/*
+    paths:
+      - "**.md"
+      - docs/**
+      - .github/CODEOWNERS
+
+env:
+  documentation_files: false
+  non_documentation_files: false
+  documentation_files_only: false
+
+jobs:
+  detect-if-skip:
+    name: Detect If Skip
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      # Check for certain paths having been modified
+      - name: Path filter
+        uses: dorny/paths-filter@v2   # See https://github.com/marketplace/actions/paths-changes-filter
+        id: path_filter
+        with:
+          filters: |
+            documentation_files:
+              - "docs/**"
+              - "**.md"
+              - ".github/CODEOWNERS"
+            non_md:
+              - "**/!(*.md)"
+            non_doc_folder:
+              - "!(docs/**)"
+            non_codeowners:
+              - "!(.github/CODEOWNERS)"
```

Comment:
This - plus how the `non_documentation_files` variable is set below -  is not a great way of adding new files to be excluded...

I would like to investigate using a bash or python script (instead of or in addition to this paths-filter action) to check for only documentation files having been changed. I think that this can be in a separate PR though, so that the filtering can still be added now.

I imagine it would look something like:
```python
edited_files = [file.txt, docs/file.md]
edited_files_filtered = edited_files.filterOut("docs/**", "**.md", ...)

if edited_files_filtered is not empty:
	# there were non-documentation files edited
	dont_skip()
else:
	skip()
```

Reply:
Hmm, maybe you could use [this action](https://github.com/marketplace/actions/changed-files) to get the full list of changed files, then you could probably use that to set your "documentation_files_only" variable in a single bash script.

Reply:
That action looks great actually. I've been messing with it a bit and I am very likely going to switch to that. Nice find!

Reply:
I've replaced the dorny/paths-filter action with this new one. The tj-actions/changed-files action has a built in boolean for checking if _only_ certain files have changed, which is exactly what I wanted.